### DISCUSSION TITLE
Error ketika menginstall requirement

### DISCUSSION THREAD

**Discussion by Auliawidyaclara2503** [teamalgoritma/community#502] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/49274058/211156931-9c853f0d-dc61-4524-badc-49d181f9d89c.png)

![image](https://user-images.githubusercontent.com/49274058/211156820-0e15904b-d1d0-4bda-8e77-3dd2eb945fa1.png)
Bagaimana agar tidak error saat menginstall requirement?

**Comment by ranydwi** [comment, marked as answer]
Dear Bu @Auliawidyaclara2503 
Untuk error yang terjadi silahkan untuk melakukan installasi requirement menggunakan code berikut:

`pip install --user -r requirements.txt`

Reference: https://paperless.readthedocs.io/en/latest/requirements.html

Semoga membantu :)
